tests/frontend/test_kibana_app/cypress/integration/step-definitions/management/decoders/navigate-to-decoders.when.js

```diff
@@ -1,19 +1,14 @@
 import { When } from 'cypress-cucumber-preprocessor/steps';
-import { getObject } from '../../../utils/driver';
+import { clickElement } from '../../../utils/driver';
 import {
-  menuListButtonsSelector,
-  subMenuListButtonsSelector,
-  wazuhButtonSelector,
+  decodersButton,
+  wazuhButton,
+  managementButton,
 } from '../../../pageobjects/wazuh-menu/wazuh-menu.page';
 
 When('The user navigates to decoders', () => {
-  getObject(wazuhButtonSelector)
-    .click();
-  getObject(menuListButtonsSelector)
-    .eq(0)
-    .click();
-  getObject(subMenuListButtonsSelector)
-    .eq(1)
-    .click();
-  cy.wait(3000);
+  clickElement(wazuhButton);
+  clickElement(managementButton);
+  clickElement(decodersButton);
+  cy.wait(5000);
```

Comment:
we should find a way to stop waiting for an arbitrary amount of time.
Im working on it, but you can try it too if you like

https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

Reply:
Yes, that's my idea, completely eliminate the 'waiting'.